crates/ethereum/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = { workspace = true }
 rust-version = { workspace = true }
 
 [dependencies]
-alloy = { version = "0.3", default-features = false, features = [
+alloy = { version = "0.12.4", default-features = false, features = [
```

Comment:
Wow! Nice bump!

Reply:
They're cutting new releases like crazy. It's hard to follow the changes though, because the release notes are way too detailed (with a list of commits).